metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/enrichment/EnrichmentClientTest.java

```diff
@@ -1,105 +0,0 @@
-package eu.europeana.enrichment.rest.client.enrichment;
-
-import static eu.europeana.metis.utils.RestEndpoints.ENRICH_ENTITY_SEARCH;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.AdditionalMatchers.not;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-
-import eu.europeana.enrichment.api.exceptions.UnknownException;
-import eu.europeana.enrichment.api.external.model.Agent;
-import eu.europeana.enrichment.api.external.model.EnrichmentBase;
-import eu.europeana.enrichment.api.external.model.EnrichmentResultBaseWrapper;
-import eu.europeana.enrichment.api.external.model.EnrichmentResultList;
-import java.net.URI;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import org.junit.jupiter.api.Test;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpMethod;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.client.RestClientException;
-import org.springframework.web.client.RestTemplate;
-
-class EnrichmentClientTest {
```

Comment:
Instead of removing it, could you have easily converted this to test the RemoteEntityResolver? They are quite related after all. Perhaps that's not possible?

Reply:
I'll see what I can do

Reply:
The conversion is not easy, we need to create some mocks and create new unit tests to do this properly.
Nevertheless, I created a new test class for `RemoteEntityResolver` and disabled the failing tests.
I created the ticket `MET-3227` (which you can see here https://europeana.atlassian.net/browse/MET-3227) with the purpose to create new unit tests for the new code created, including `RemoteEntityResolverTest`